### DISCUSSION TITLE
Thinq1 vs Thinq2

### DISCUSSION THREAD

**Discussion by iShaymus** [nVuln/homebridge-lg-thinq#21] [Q&A]
How can we tell the difference between a ThinQ1 device and a ThinQ2 device.  My refrigerator is loaded in the ThinQ app but it does not load as a device in HomeBridge.  I'm pretty sure that I'm correctly authenticated as I'm not getting auth errors.

**Comment by nVuln** [comment]
please turn on homebridge debug mode, you can see your device data in debug logging
something like this `"platformType":"thinq2"`
